### PR TITLE
Feature/training data pipeline

### PR DESCRIPTION
**Pull request by TashaSkyUp**
# Description

This pull request is for the training data pipeline; This consists of a GUI style application (CLI to come) that allows for great control and far better results when preparing data for training on custom models.

--- Loom Video ---
https://www.loom.com/share/2f6741ce6470402fa19bf59f6952340d

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests) ** --- _Includes some tests_ --- **
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: data_management/story_photo_color_transformations.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+        self.redraw()
+
+
+if __name__ == "__main__":
+    root = tk.Tk()
+    # Resize the display window
+    root.geometry("800x1000")
+    app = Application(master=root, next_phase=None)
+    app.mainloop()
```
Comment (peterger8y) [inline data_management/story_photo_color_transformations.py:155]: Very impressive code blocks here. Hopefully I can use some fo it for snippet preprocessing. Either way, Good job to you and your Team!

**Review by peterger8y — APPROVED** [review]
Very nice! this should help the training pipeline quite a bit!

**Review by BritVandy — APPROVED** [review]
Amazing job! Thank you for all of the documentation.  Everything is well explained. I’m excited to see this implemented!

**Review by techthumb1 — APPROVED** [review]
Very nice, really great work! Code is well written, well commented, easy to follow. My only critique would be to include some docstrings here and there, but again very nice.

**Comment by tinomen** [comment]
This is a massive PR and deserves scrutiny. I'm surprised by the lack of inline comments. This is a common problem with large PRs, it's too big to hold in our heads so it's much harder to review line by line. Without evidence of a detailed review, it will make the approving comments equivalent to 🤞🏻 

Kudos to @peterger8y for the inline comment.

@techthumb1 in the future do not hold back. Add inline comments in places you think deserve documentation (docstrings).

**Comment by singparvi** [comment]
Great work Tasha. I reviewed your code during the collaboration with you. Your work is clean and I liked your idea of importing code as a package to be used in any file. This was very helpful in making multiple files that were manageable and easy to understand.

**Pull request merged**